en/components/themes/sass/palettes.md

```diff
@@ -0,0 +1,307 @@
+    background: $my-primary-600;
+    border-color: $my-primary-A700;
+}
+
+.warning-bg {
+    background: $my-warning-color;
+}
+```
+
+If you omit the `$palette` argument, the value stored in `$default-palette` is used. If you do not provide `$color` and/or `$variant`, they will be assigned to `primary` and `500` respectively.
+
+<div class="divider"></div>
+
+## Contrast Text Colors
+
+Similar to how we retrieve color variants, there's a way to get the contrast text color for each of the color variants in the palette.
+
+```scss
+$my-primary-800: igx-color($my-palette, 'primary', 600);
+$my-primary-800-text: igx-contrast-color($my-palette, 'primary', 600);
+
+// sample usage
+.my-awesome-article {
+    color: $my-primary-800-text;
+    background: $my-primary-800;
+}
+```
+
+<div class="divider"></div>
+
+## Color Classes
+
+We understand that some people prefer to use CSS classes to apply colors to web elements, be it text or background. We have a mixin that allows you to generate CSS classes for each color in the palette.
+
+The mixin takes a few arguments that allow you control the CSS class name as well as the CSS property to pass the palette color to;
+
+- `$prop` - The CSS property to assign the color to.
+- `$prefix` - A prefix string to be attached to the generated class name. The default is 'igx'.
+- `$suffix` - A suffix string to be attached to the generated class name.
+- `$palette` - The palette to use.
+
+For instance, if you want to generate CSS classes that apply background color to elements, you can do the following:
+
+```scss
+@include igx-color-classes(
+    $prop: 'background-color',
+    $prefix: 'bg'
+);
+```
+The above code will generate CSS classes for each color variant in the palette. For instance, the `500` color variant of the `primary` palette will be given the following class `.bg-primary-500`;
+
+```html
+<div class="bg-primary-500">...</div>
+```
+
+<div class="divider--half"></div>
+
+## CSS Variables
+
+When reading about the color palette in the [CSS Variables](../palettes.md) section of the documentation, you would've noticed that all palette colors are included as CSS variables. We do this internally every time we generate a theme using the `igx-theme` mixin. The `igx-theme` calls another mixin in its body - `igx-palette-vars`. It takes a palette and converts the colors in it into CSS variables. 
+
+You use this mixin when you want your custom palette colors to be included as CSS variables.
+
+Here's an example:
+
+```scss
+$my-palette: igx-palette(
+  $primary: #2ab759,
+  $secondary: #f96a88,
+);
+
+@include igx-palette-vars($my-palette);
+```
+
+## API Reference
+* [Palettes]({environment:sassApiUrl}/index.html#function-igx-palette)
+* [Getting Palette Colors]({environment:sassApiUrl}/index.html#function-igx-color)
+* [Getting Contrast Colors]({environment:sassApiUrl}/index.html#function-igx-contrast-color)
+* [Generating Color Classes]({environment:sassApiUrl}/index.html#mixin-igx-color-classes)
```

Comment:
The last link is wrong

Reply:
This is related to the `@access` annotation in the source documentation in Ignite UI for Angular. It is set to private.